scripts/pages/map-selector/map-entry.js

```diff
@@ -28,84 +28,73 @@ class MapEntry {
 		const mapData = $.GetContextPanel().mapData;
```

Comment:
`$.GetContextPanel()` has to call into C++ every time which is nice to avoid. In this case I would just do 
```js
const { mapData, userMapData, isDownloading } = $.GetContextPanel()
```